lib/app/features/home/match_details/match_details_tabs/match_summary_tab/elements/halves_events_display.dart.dart

```diff
@@ -21,99 +21,78 @@ class HalvesEventsDisplay extends StatelessWidget {
         borderRadius: BorderRadius.circular(12),
         color: AppColors.listTileGrey,
       ),
-      height: 200,
-      child: Row(
+      child: ListView(
+        shrinkWrap: true,
+        primary: false,
         children: [
-          Expanded(
-            flex: 1,
-            child: Column(
-              children: [
-                SizedBox(
-                  height: 200,
-                  child: ListView(
-                    physics: const NeverScrollableScrollPhysics(),
-                    children: [
-                      if (isHalfTime == true)
-                        for (final event in matchEvents) ...[
-                          if (event.team?.id != null && event.time?.elapsed != null && event.player?.name != null)
-                            if (homeTeamID == event.team?.id && event.time!.elapsed! < 45)
-                              Padding(
-                                padding: const EdgeInsets.only(left: 8.0, top: 8.0),
-                                child: EventTextTemplate(
-                                    leadingProperty:
-                                        event.time?.elapsed.toString() ?? AppConstVariables.stringPlaceholder,
-                                    player: event.player?.name ?? AppConstVariables.stringPlaceholder,
-                                    isHomeTeam: true),
-                              )
-                            else
-                              const SizedBox.shrink(),
-                        ]
-                      else
-                        for (final event in matchEvents) ...[
-                          if (event.team?.id != null && event.time?.elapsed != null && event.player?.name != null)
-                            if (homeTeamID == event.team?.id && event.time!.elapsed! > 45)
-                              Padding(
-                                padding: const EdgeInsets.only(left: 8.0, top: 8.0),
-                                child: EventTextTemplate(
-                                    leadingProperty:
-                                        event.time?.elapsed.toString() ?? AppConstVariables.stringPlaceholder,
-                                    player: event.player?.name ?? AppConstVariables.stringPlaceholder,
-                                    isHomeTeam: true),
-                              )
-                            else
-                              const SizedBox.shrink(),
-                        ]
-                    ],
-                  ),
-                )
-              ],
-            ),
+          Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Expanded(
+                flex: 1,
```

Comment:
Not an error, you just don't need to specify `flex` if you set it to 1. It's the default value of `Expanded` Widget

Reply:
you're right, must have put that one automatically while creating display, removed